src/main/java/com/polyscripts/contactManagement/models/ContactFreelance.java

```diff
@@ -0,0 +1,25 @@
+package com.polyscripts.contactManagement.models;
+
+import lombok.Getter;
+import lombok.Setter;
```

Comment:
there is no needs to create a freelancer entity, a freelancer is a contact that don't have a tva numbern that's all